src/app/pages/WelcomePage/WelcomePage.tsx

```diff
@@ -0,0 +1,31 @@
+import WelcomePageIcon from '../../components/Welcome/WelcomePageIcon.svg';
+import { FormEvent, useState } from 'react';
+import styles from './WelcomePage.module.css';
+import Button from '../../components/Button/Button';
+
+function WelcomePage(): JSX.Element {
+  const [name, setName] = useState('');
+
+  function handleSubmit(event: FormEvent) {
+    event.preventDefault();
+  }
+
+  return (
+    <div className={styles.container}>
+      <img className={styles.icon} src={WelcomePageIcon} alt="" />
```

Comment:
Is Your WelcomePageIcon the "dear diary" text? Then you should add an `alt` to display, if image loading fails.

Reply:
Good Idea! I didn't think about that. Thanx!